packages/react-instantsearch-core/src/connectors/__tests__/connectAutoComplete.js

```diff
@@ -13,38 +13,79 @@ describe('connectAutoComplete', () => {
         { contextValue },
         {},
         {
-          results: { hits },
+          results: { hits, page: 0, hitsPerPage: 20 },
         }
       );
       expect(props).toEqual({
-        hits,
+        hits: [{ __position: 1 }],
         currentRefinement: '',
       });
 
       props = connect.getProvidedProps(
         { contextValue },
         { query: 'query' },
         {
-          results: { hits },
+          results: { hits, page: 0, hitsPerPage: 20 },
         }
       );
       expect(props).toEqual({
-        hits,
+        hits: [{ __position: 1 }],
         currentRefinement: 'query',
       });
 
       props = connect.getProvidedProps(
         { defaultRefinement: 'query', contextValue },
         {},
         {
-          results: { hits },
+          results: { hits, page: 0, hitsPerPage: 20 },
         }
       );
       expect(props).toEqual({
-        hits,
+        hits: [{ __position: 1 }],
         currentRefinement: 'query',
       });
     });
+
+    it('provides current hits to the component with extra info', () => {
```

Comment:
We could be more descriptive for the test name.

Reply:
```suggestion
    it('provides current hits to the component with queryID & position', () => {
```